ansible/group_vars/wakuv2-test.yml

```diff
@@ -35,3 +35,11 @@ open_ports_list:
   - { port: '{{ nim_waku_websockify_cont_port }}', protocol: 'tcp' }
   - { port: '80', comment: 'Certbot verification', protocol: 'tcp' }
   - { port: '{{ nim_waku_metrics_port }}',         protocol: 'tcp', chain: 'VPN', ipset: 'metrics.hq' }
+
+#Enable websockets in waku
+nim_waku_websocket_enabled: true
+nim_waku_websocket_cont_port: 8000
+nim_waku_websocket_domain: '{{ dns_entry }}'
+nim_waku_websocket_ssl_dir: '/etc/letsencrypt'
+nim_waku_websocket_ssl_cert: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain }}/fullchain.pem'
+nim_waku_websockify_ssl_key: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain }}/privkey.pem'
```

Comment:
I'd just copy the values from `nim_waku_websockify_ssl_*` variables.

Reply:
websockify needs to removed later , its better to separate it out.

Reply:
Yes, it should be separate, also, you left one variable unchanged: `nim_waku_websockify_ssl_key`

I will adjust your PR and rebase it and squash it.